src/types/utils.d.ts

```diff
@@ -1,7 +1,34 @@
-import { Client } from '../client'
+// import { Client } from '../client'
```

Comment:
This could be removed.

Reply:
Oh yeah I forgot to follow up on that... I also commented out `TestRequestPayload` but that got uncommented by my VS code. I guess we don't need that import, since that type is used in `runTestCase` which is used by pretty much every test... right?

Reply:
Confirmed that the determinism tests run (which reference the type directly) fine with it commented out:

```
 ✓ src/__test__/e2e/signing/determinism.test.ts (50) 257847ms

Test Files  1 passed (1)
     Tests  50 passed (50)
      Time  260.38s (in thread 257.85s, 100.98%)
      ```

Reply:
[457a106](https://github.com/GridPlus/gridplus-sdk/pull/494/commits/457a106ccc3ba240bdcea4375e0df828e3b7e54e)